components/shared_code/src/shared_data_model/logos/__init__.py

```diff
@@ -1,5 +1,14 @@
-"""Define this as the path to the logos."""
+"""Logos for sources."""
 
-import pathlib
+import importlib.resources
 
-LOGOS_ROOT = pathlib.Path(__file__).parent.absolute()
+
+def get_logo(name: str) -> bytes:
+    """Return the logo."""
+    return importlib.resources.read_binary("shared_data_model.logos", f"{name}.png")
```

Comment:
Why not `("shared_data_model", "logos", f"{name}.png")`; shouldn't it be a `pathlib.Path`?

Reply:
No, it can be a module name, see https://docs.python.org/3/library/importlib.resources.html#importlib.resources.Anchor.